tests/testUtils.py

```diff
@@ -244,18 +245,18 @@ def writeCSV(self, data, fileName):
 
     {'id1' : {'f1':n1, 'f2':n2}, 'id2' : {'f1':n3, 'f2':n4}}
     """
-    csvFile = open(fileName, 'wb')
+    csvFile = open(fileName, 'w')
     csvFileWriter = csv.writer(csvFile)
     # get the headers from the first row
-    header = sorted(data[data.keys()[0]].keys())
+    header = list(data[list(data.keys())[0]].keys())
```

Comment:
@jcfr, I'm rewriting this a bit, as it is a bit buggy if you e.g. only test the 'breast1' test case. In that case, no results are written as the first test case ('brain1') does not contain any calculated features. The test cases are based on the baseline. I'll rewrite it so, that only the test cases that have been actually tested are written to the result and diff files, with no files written if no tests were actually run.